app/src/main/java/com/example/sun/data/repository/CurrentWeatherRepository.kt

```diff
@@ -16,6 +16,15 @@ class CurrentWeatherRepository(
     ) {
         remote.getCurrentWeather(listener, city)
     }
+
+    override fun getCurrentLocationWeather(
+        listener: OnResultListener<CurrentWeather>,
+        lat: Double,
+        lon: Double,
+    ) {
```

Comment:
```suggestion
    override fun getCurrentLocationWeather(
        listener: OnResultListener<CurrentWeather>,
        latitude: Double,
        lontitude: Double,
    ) {
```
Nên đặt tên đồng nhất trong dự án